osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -74,10 +73,33 @@ private void load(Storage storage, OsuConfigManager config, OsuGame game)
                     Action = () => storage.PresentExternally(),
                 });
 
+                Add(new SettingsButton
+                {
+                    Text = "Compress log files",
```

Comment:
```suggestion
                    Text = "Export log files",
```

`Compress` makes me thing that the logs will be compressed to save space to disk. I wouldn't expect this button to save the logs and open a explorer window to them.